tests/rptest/clients/rpk.py

```diff
@@ -432,6 +432,18 @@ def sasl_create_user(self,
 
         return self._run(cmd)
 
+    def sasl_list_users(self):
```

Comment:
nit: I'd change the commit message to start with `test/...` or just `test` , makes it more immediately obvious that this is a commit that just adds/modifies a test

Reply:
Makes sense. I've renamed the commit now to [test: rpk user delete of complex username](https://github.com/redpanda-data/redpanda/pull/16694/commits/894242bdc9597367b9f95fdbfee9a4ab812f26bd).